### ISSUE TITLE
Error sharing directory

### ISSUE THREAD

**Issue by Dominiquini** [mifi/ezshare#29]
I tried to share my F: directory, but not is showing in the browser. The console showed this error:

Error: EBUSY: resource busy or locked, lstat 'F:\DumpStack.log.tmp'

I run the application this way: "ezshare --port 8008 F:\"

OS: Windows 10 x64
App Version: 1.8.3

Thanks.

**Comment by Dominiquini** [comment]
Yes, I'm using Command Prompt without administrator privileges...

I can't delete this file because it is protected (I don't have permissions).
The application breaks every time it finds a file of this type (it doesn't even list the other files in the folder).
The application works normally if there is no protected file in the folder!

I suggest that if ezshare finds a file that it cannot read, it simply skips it and lists the other files and directories! 

Thanks.

**Comment by mifi** [comment]
try the nwest version now
